### DISCUSSION TITLE
I don't know which model is right for me.

### DISCUSSION THREAD

**Discussion by kbuwel** [nomadkaraoke/python-audio-separator#135] [Q&A] [answered]
There are many models available in audio-separator.
But I don't know which model to work with.
I downloaded and worked with several Roformer Models, but I don't think I'm satisfied with the results.
I can remove background music, but not sound effects.
Is there a way to know which model is right for me?
When I download and run a model, it takes a long time and takes up a lot of space.
It would be nice if there was a site that explains which model can remove which instrument or sound effect.

**Comment by beveradb** [comment, marked as answer]
I literally just posted that 😅

https://github.com/nomadkaraoke/python-audio-separator/discussions/133#discussion-7314015

**Comment by kbuwel** [comment]
I think there was a misunderstanding.
I need an article or a site link that explains what instruments or sound effects the models can remove.
https://mvsep.com/en
I went to the site but there were hardly any models used in audio-separator.

**Reply by beveradb** [reply]
How does my post not explain that? I don't really understand what you're looking for, but you're welcome to create a post yourself if you think you can do better than what I wrote 😊

**Comment by kbuwel** [comment]
Google translation is not good at expressing my thoughts.
Swear Google.
I just want to know which model can remove which instruments or sound effects.
I keep saying the same thing again and again.
I don't know which model can remove which sound.
I'd be a developer if I knew that.

**Reply by beveradb** [reply]
If you aren't a developer why are you on GitHub? 😅
This is a tool which assumes technical understanding; if you aren't technical you are probably better off sticking with https://ultimatevocalremover.com

**Comment by beveradb** [comment]
I'm sorry the language barrier makes things extra difficult but I'm not sure how else I can help you. There must still be some sort of misunderstanding here, but I'm not sure what. 

In the post I wrote here: 
https://github.com/nomadkaraoke/python-audio-separator/discussions/133#discussion-7314015

There is a section titled "What you are trying to achieve".

In that section, I list all of the models I recommend for the different purposes served by `audio-separator`.

For example, "htdemucs_6s.yaml (this produces 6 stems: Drums, Bass, Guitar, Piano, Other, Vocals)". Or `Kim_Vocal_2.onnx` which separates vocals. Or `kuielab_a_bass.onnx` which separates just bass.

"which model can remove which sound" - that is explained as best as I am able to in the post I wrote.

Perhaps you think there is some other information I am withholding from you, but I promise there is not!

Can you explain what your use case is, what input audio you are using, and what you are hoping for?

**Comment by kbuwel** [comment]
Andrew Beveridge와는 통신이 불가능하다고 생각했습니다.

**Comment by oracle-999** [comment]
Which model is right for you?that depends on what you want literally. Seperation tech now could only seperate simple elements, mixed drums,mixed bass,and vocal ,surely mixed too.extracting the original dry sound would be almost impossible...or the algorithm is  just too advanced
